libs/dmlf/include/dmlf/colearn/muddle_learner_networker_impl.hpp

```diff
@@ -40,36 +40,39 @@ namespace colearn {
 class MuddleLearnerNetworkerImpl : public AbstractMessageController
 {
 public:
-  using Taskpool                      = oef::base::Taskpool;
-  using Threadpool                    = oef::base::Threadpool;
-  using TaskP                         = Taskpool::TaskP;
+  using Taskpool = oef::base::Taskpool;
+
+  using Address                       = fetch::muddle::Address;
+  using AlgorithmClass                = AbstractMessageController::AlgorithmClass;
```

Comment:
just curious, does the Classiness matter for the naming convention?

Reply:
Not my selection of name tbh.